.cirrus.yml

```diff
@@ -41,22 +41,25 @@ task:
     apt-get update 
     apt-get install --yes nodejs 
     node --version && npm --version
-
   # Info about Cirrus CI caching see: https://cirrus-ci.org/guide/writing-tasks/#cache-instruction
   node_modules_cache:
     folder: /home/ranchertest/node_modules
     fingerprint_script: cat package-lock.json
     populate_script:
       # Passing DEBUG=pw:install in order to verify if all Playwright deps were installed properly
-      - sudo --login --user=ranchertest DEBUG=pw:install npm ci 
-      - sudo --login --user=ranchertest ranchertest ./node_modules/.bin/playwright install-deps
+      - ${SUDO_NON_ROOT_CMD} DEBUG=pw:install npm ci
+      - ${SUDO_NON_ROOT_CMD} ranchertest ./node_modules/.bin/playwright install-deps
+
+  # Ensure the tools were installed properly
+  post_install_script:
+    - ${SUDO_NON_ROOT_CMD} node scripts/postinstall.mjs
```

Comment:
Unfortunately running npm ci in `node cache` does not upload the postinstall tools, required for the tests.
Created another step to ensure the `postinstall` will install all required tools properly.